src_pop/core/biogeochem/POP.F90

```diff
@@ -4,13 +4,51 @@
 ! Written by Vanessa Haverd, Ben Smith and Lars Nieradzik
 ! Report Bugs to Vanessa.Haverd@csiro.au
 
+!*
+! This MODULE contains code for the woody demography model POP. The principal components
+! of this MODULE are the initialisation SUBROUTINE [[InitPOP2D_Poisson]], time stepping
+! SUBROUTINE [[POPStep]] and routines for allometry, recruitment, mortality and disturbance.
+! The main SUBROUTINE is [[POPStep]] which is called from SUBROUTINE POPdriver in casa_cable.F90.
+! This SUBROUTINE calls most other subroutines and functions in POP.
+! POP receives StemNPP from CASA and returns woody turnover rate and vegetation height to the
+! main model. External inputs is disturbance regime (total or total and partial) and disturbance interval.
+! Main variables and their units are:
+! Inputs:
```

Comment:
I'd include a blank line here to get this onto a new line

Reply:
I have added a few more blank lines. Looks like there is a tradeoff between readability of the code and readability of the rendered documentation.